docs/NMOS With H.264.md

```diff
@@ -58,11 +58,116 @@ The term 'strict-CBR' corresponds to the definition of constant bit-rate at sect
+- H.264 Flow resources MAY be associated with many Senders at the same time through the Senders' `flow_id` attributes. The H.264 Flow is therefore unaffected by the use of a specific transport. 
+- H.264 Flow resources MAY be associated with many Flows at the same time through the Flows' `parents` attributes. The H.264 Flow is therefore unaffected by being the parent of some other Flows.
+
+For Nodes implementing IS-04 v1.3 or higher, the following additional requirements on the Flow resource apply.
+
+In addition to those attributes defined in IS-04 for all coded video Flows, the following attributes defined in the [Flow Attributes Register](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/) of the [NMOS Parameter Registers][] are used for H.264.
+
+These attributes provide information for Controllers and Users to evaluate stream compatibility between Senders and Receivers.
+
+- [Components](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#components)
+  The Flow resource MUST indicate the color (sub-)sampling, width, height and depth of the associated uncompressed picture using the `components` attribute. The `components` array values MUST correspond to the stream's active parameter sets values. A Flow MUST track the stream's current active parameter sets.
+
+Informative note: ST 2110-22 does not require the `sampling` or `depth` SDP parameters. RFC 6184 does not define any such SDP parameters. The `sampling` and `depth` of the associated uncompressed picture could be derived from the H.264 active parameter sets by a Receiver.
+
+- [Profile](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#profile)
+  The Flow resource MUST indicate the H.264 profile, which defines algorithmic features and limits that SHALL be supported by all decoders conforming to that profile. It SHALL comply with the stream's active parameter sets. The permitted `profile` values are strings, defined as per ITU-T  Rec. H.264 Annex A
+
+  - "BaselineConstrained"
+  - "Baseline" (Default if not specified in the SDP transport file)
+  - "Main"
+  - "Extended"
+  - "High"
+  - "HighProgressive"
+  - "HighConstrained"
+  - "High10"
+  - "High10Progressive"
+  - "High-422"
+  - "HighPredictive-444"
+  - "High10Intra"
+  - "HighIntra-422"
+  - "HighIntra-444"
+  - "CAVLCIntra-444"
+
+Informative note: The names of the profiles in string form have been derived from the names used at Annex A of the H.264 standard with whitespace omitted, the sampling mode always positioned at the end of the string, preceded by a '-' and the terms High and Baseline positioned at the beginning of the string.
+
+The [NMOS Parameter Registers] is the official reference for the profiles strings. In the case of discrepancies between this specification and the [NMOS Parameter Registers], the registers have precedence.
```

Comment:
The Parameter Registers occupy a slight grey area in that they are not formally an AMWA specification. Also, I believe we should strive not to CHANGE any of these values, but may perhaps add new ones?

```suggestion
The profile strings in this specification are included in the [NMOS Parameter Registers][]. Additional strings may be added there in the future.
```

Reply:
done